test/infra/infra-dbdeployer-mysql90/docker-compose-init.bash

```
@@ -0,0 +1,153 @@
+#!/bin/bash
+# RELIABLY CAPTURE INFRA_ID FROM ENVIRONMENT OR DIRECTORY NAME
+if [ -z "${INFRA_ID}" ]; then
+    export INFRA_ID=$(basename $(dirname $(pwd)) | sed 's/infra-//; s/docker-//')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)